pkg/cmd/admin/admin.go

```diff
@@ -45,54 +47,55 @@ func NewCommandAdmin(name, fullName string, out io.Writer, errout io.Writer) *co
 
 	groups := templates.CommandGroups{
 		{
-			Message: "Basic Commands:",
+			Message: "Component Installation:",
+			Commands: []*cobra.Command{
+				router.NewCmdRouter(f, fullName, "router", out),
+				exipfailover.NewCmdIPFailoverConfig(f, fullName, "ipfailover", out, errout),
+				registry.NewCmdRegistry(f, fullName, "registry", out),
+			},
+		},
+		{
+			Message: "Security and Policy:",
 			Commands: []*cobra.Command{
 				project.NewCmdNewProject(project.NewProjectRecommendedName, fullName+" "+project.NewProjectRecommendedName, f, out),
 				policy.NewCmdPolicy(policy.PolicyRecommendedName, fullName+" "+policy.PolicyRecommendedName, f, out, errout),
 				groups.NewCmdGroups(groups.GroupsRecommendedName, fullName+" "+groups.GroupsRecommendedName, f, out),
+				cert.NewCmdCert(cert.CertRecommendedName, fullName+" "+cert.CertRecommendedName, out, errout),
+				admin.NewCommandOverwriteBootstrapPolicy(admin.OverwriteBootstrapPolicyCommandName, fullName+" "+admin.OverwriteBootstrapPolicyCommandName, fullName+" "+admin.CreateBootstrapPolicyFileCommand, out),
 			},
 		},
 		{
-			Message: "Install Commands:",
+			Message: "Node Management:",
 			Commands: []*cobra.Command{
-				router.NewCmdRouter(f, fullName, "router", out),
-				exipfailover.NewCmdIPFailoverConfig(f, fullName, "ipfailover", out, errout),
-				registry.NewCmdRegistry(f, fullName, "registry", out),
+				admin.NewCommandNodeConfig(admin.NodeConfigCommandName, fullName+" "+admin.NodeConfigCommandName, out),
+				node.NewCommandManageNode(f, node.ManageNodeCommandName, fullName+" "+node.ManageNodeCommandName, out, errout),
+				cmdutil.ReplaceCommandName("kubectl", fullName, kubectl.NewCmdCordon(f.Factory, out)),
```

Comment:
I'd probably rename manage node then.  Might be ok for admin commands.

On Wed, Jul 6, 2016 at 3:24 PM, David Eads <notifications@github.com> wrote:

> In pkg/cmd/admin/admin.go
> <https://github.com/openshift/origin/pull/9578#discussion_r69795423>:
>
> >  			Commands: []*cobra.Command{
> > -				router.NewCmdRouter(f, fullName, "router", out),
> > -				exipfailover.NewCmdIPFailoverConfig(f, fullName, "ipfailover", out, errout),
> > -				registry.NewCmdRegistry(f, fullName, "registry", out),
> > +				admin.NewCommandNodeConfig(admin.NodeConfigCommandName, fullName+" "+admin.NodeConfigCommandName, out),
> > +				node.NewCommandManageNode(f, node.ManageNodeCommandName, fullName+" "+node.ManageNodeCommandName, out, errout),
> > +				cmdutil.ReplaceCommandName("kubectl", fullName, kubectl.NewCmdCordon(f.Factory, out)),
>
> I expected these as oadm node cordon as opposed to oadm cordon. Seems
> like these are directly related to nodes.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/openshift/origin/pull/9578/files/4cb6166306752d826a174e8249af5a9233796705..472edee18a4fef25d07d95a75b28724550754420#r69795423>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ABG_p0NDHnO-wBLRdv65CYcj55zJu6-Qks5qTAEFgaJpZM4I-xTE>
> .
>